backend/api/EventHandlers/MissionScheduler.cs

```diff
@@ -41,7 +41,6 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
                     }
                     if (
                         freshMission.Robot.Status is not RobotStatus.Available
-                        || freshMission.StartTime > DateTimeOffset.UtcNow
```

Comment:
Have you tested if the order of missions is maintained?

So if you schedule multiple missions (three or more) and we schedule them one at a time again and again and again in the same order. Do we know that the missions will be executed in the correct order? Now the UpcomingMissions object is collected from the database but I'm not sure if the order is guaranteed to be the same.

Reply:
I think we should keep this line to keep scheduling functionality in backend

Reply:
The order of the missions is not maintained as is either